src/pages/workspace/accounting/netsuite/export/NetSuiteInvoiceItemPreferenceSelectPage.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import type {ValueOf} from 'type-fest';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import SelectionScreen from '@components/SelectionScreen';
+import type {SelectorType} from '@components/SelectionScreen';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections/NetSuiteCommands';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@navigation/Navigation';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import * as Policy from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type MenuListItem = ListItem & {
+    value: ValueOf<typeof CONST.NETSUITE_INVOICE_ITEM_PREFERENCE>;
+};
+
+function NetSuiteInvoiceItemPreferenceSelectPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '-1';
+    const config = policy?.connections?.netsuite.options.config;
+
+    const {items} = policy?.connections?.netsuite.options.data ?? {};
+
+    const selectedItem = useMemo(() => {
+        const selectedRec = (items ?? []).find((item) => item.id === config?.invoiceItem);
+        return selectedRec;
+    }, [items, config?.invoiceItem]);
+
+    const data: MenuListItem[] = Object.values(CONST.NETSUITE_INVOICE_ITEM_PREFERENCE).map((postingPreference) => ({
+        value: postingPreference,
+        text: translate(`workspace.netsuite.invoiceItem.values.${postingPreference}.label`),
+        keyForList: postingPreference,
+        isSelected: config?.invoiceItemPreference === postingPreference,
+    }));
+
+    const selectInvoicePreference = useCallback(
+        (row: MenuListItem) => {
+            if (row.value !== config?.invoiceItemPreference) {
+                Connections.updateNetSuiteInvoiceItemPreference(policyID, row.value, config?.invoiceItemPreference);
+            }
+            if (row.value === CONST.NETSUITE_INVOICE_ITEM_PREFERENCE.CREATE) {
```

Comment:
I don't have the context of this change. So if `invoiceItemPreference` is select then we don't navigate?